.github/workflows/ci.yml

```diff
@@ -154,7 +154,7 @@ jobs:
             cd matic-cli/devnet/code/contracts
             npm run truffle exec scripts/deposit.js -- --network development $(jq -r .root.tokens.MaticToken contractAddresses.json) 100000000000000000000
             cd -
-            bash bor/integration-tests/smoke_test.sh
+            timeout 10m bash bor/integration-tests/smoke_test.sh
```

Comment:
I'd increase the potential timeout to 20m just to be sure, as I've seen myself sometime this getting up to roughly 18mins. It won't anyway have an impact when we actively check for completion

Reply:
[b2a453b](https://github.com/maticnetwork/bor/pull/583/commits/b2a453b115d1bcb4b4a63b291f8ce9c41224e4f0)